cache/sampleStoreInit.js

```diff
@@ -137,7 +156,7 @@ function populate() {
   const msg = 'Populating redis sample store from db';
   console.log(msg); // eslint-disable-line no-console
 
-  const promises = [populateSamples(), populateAspects()];
+  const promises = [populateSamples(), populateAspects(), populateSubjects()];
```

Comment:
This doesn't matter at an implementation/execution level here, but I think subjects and aspects should be listed in this array *before* samples--it (very subtly) helps communicate that subjects and aspects have to exist for a sample to exist